core/controller/init.sh

```diff
@@ -1,6 +1,11 @@
 #!/bin/bash
 
+if [[ $( ls /conf/jmxremote.* 2> /dev/null ) ]]
+then
+  mv /conf/jmxremote.* /root ; chmod 600 /root/jmxremote.*
```

Comment:
Woukd it make sense to create a new script in the scala container, that handles these jmx-files?
If doing it there, we would avoid the code duplication in the init script in controller and invoker.

Reply:
As suggested I have moved the common functionality into a  file that is copied into the Scala container